Doesn't build on alpine linux (unknown type `int64_t`)

Hey! I tried to build this on alpine linux but got this error:

```
../src/cre2.h:65:60: error: unknown type name 'int64_t'
```

and a few more of these. I applied a simple patch: https://gist.github.com/chmanie/e0d695b148b252723257eb204e53ace3

While this works for me, I don't know how this could affect other platforms, so I'm hoping to start a conversation on how to fix this for good.

--- a/src/cre2.h
+++ b/src/cre2.h
@@ -21,6 +21,8 @@
 extern "C" {
 #endif
 
+#include <stdint.h>
+
 #ifndef cre2_decl
 #  define cre2_decl	extern
 #endif

